### ISSUE TITLE
Failed at the meteor-dev-bundle@0.0.0 install script 'node npm-rebuild.js'.    

### ISSUE THREAD

**Issue by lpender** [AdmitHub/meteor-buildpack-horse#168]
##### Type of problem

 - Failure to build

##### Summary

`git push heroku master` 

`Failed at the meteor-dev-bundle@0.0.0install script 'node npm-rebuild.js'`

##### Contents of `.meteor/release`

```
METEOR@1.4.3.2
```

##### Buildpack version

```
https://github.com/lpender/meteor-buildpack-horse
```

##### Logs

```
remote: Compressing source files... done.        
remote: Building source:        
remote: 
remote: -----> Node.js app detected        
remote: + dirname -- /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/bin/compile        
remote: + cd -P -- /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/bin        
remote: + cd ..        
remote: + pwd -P        
remote: + BUILDPACK_DIR=/app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb        
remote: + APP_CHECKOUT_DIR=/tmp/build_94dcde2cf266af783b2ac6f88761a960        
remote: + APP_SOURCE_DIR=/tmp/build_94dcde2cf266af783b2ac6f88761a960        
remote: + [ -n  ]        
remote: + [ ! -d /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor ]        
remote: + [ ! -d /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor ]        
remote: + mktemp -d /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-XXXX        
remote: + METEOR_DIR=/app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr        
remote: + COMPILE_DIR_SUFFIX=.meteor/heroku_build        
remote: + COMPILE_DIR=/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build        
remote: + METEOR_PRETTY_OUTPUT=0        
remote: + mkdir -p /tmp/build_94dcde2cf266af783b2ac6f88761a960 /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build        
remote: + PATH=/app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr/.meteor:/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/bin:/app/bin:/app/vendor/bundle/bin:/app/vendor/bundle/ruby/2.4.0/bin:vendor/yarn-0.22.0/bin/:/usr/local/bin:/usr/bin:/bin:/tmp/codon/vendor/bin        
remote: + [ -z https://litmixes.herokuapp.com ]        
remote: + [ -z https://litmixes.herokuapp.com ]        
remote: + echo -----> Installing node        
remote: -----> Installing node        
remote: + curl -sS --get https://semver.io/node/resolve/4.10.x        
remote: + NODE_VERSION=7.7.4        
remote: + NODE_URL=http://s3pository.heroku.com/node/v7.7.4/node-v7.7.4-linux-x64.tar.gz        
remote: + + curl -sS http://s3pository.heroku.com/node/v7.7.4/node-v7.7.4-linux-x64.tar.gz -o -        
remote: tar -zxf - -C /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build --strip 1        
remote: + export npm_config_prefix=/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build        
remote: + export CPATH=/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/include        
remote: + export CPPPATH=/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/include        
remote: + echo -----> Installing meteor        
remote: -----> Installing meteor        
remote: + HOME=/app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr /bin/sh        
remote: + curl -sS https://install.meteor.com        
remote: Downloading Meteor distribution        
remote:         
remote: Meteor 1.4.3.2 has been installed in your home directory (~/.meteor).        
remote: 
remote: Now you need to do one of the following:        
remote: 
remote:   (1) Add "$HOME/.meteor" to your path, or        
remote:   (2) Run this command as root:        
remote:         cp "/app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr/.meteor/packages/meteor-tool/1.4.3_2/mt-os.linux.x86_64/scripts/admin/launch-meteor" /usr/bin/meteor        
remote: 
remote: Then to get started, take a look at 'meteor --help' or see the docs at        
remote: docs.meteor.com.        
remote: + METEOR=/app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr/.meteor/meteor        
remote: + + sed -e s/Meteor /METEOR@/        
remote: HOME=/app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr/.meteor/meteor --version        
remote: + CUR_RELEASE=METEOR@1.4.3.2        
remote: + cat /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/release        
remote: + APP_RELEASE=METEOR@1.4.2.3        
remote: + test METEOR@1.4.3.2 != METEOR@1.4.2.3        
remote: + echo METEOR@1.4.3.2\nMETEOR@1.4.2.3        
remote: + head -n1        
remote: + sort --version-sort        
remote: + OLDER_RELEASE=METEOR@1.4.2.3        
remote: + [ METEOR@1.4.3.2 = METEOR@1.4.2.3 ]        
remote: + echo -----> Bundling bundle        
remote: -----> Bundling bundle        
remote: + cd /tmp/build_94dcde2cf266af783b2ac6f88761a960        
remote: + echo -----> Checking if this meteor version supports --server-only        
remote: -----> Checking if this meteor version supports --server-only        
remote: + set +e        
remote: + + HOME=/app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr/.meteor/meteor help build        
remote: grep -e --server-only        
remote: + HAS_SERVER_ONLY=Usage: meteor build <output path> [--debug] [--directory] [--server-only]        
remote: Pass `--server-only` to skip building mobile apps, but still build the        
remote:   --server-only       Skip building mobile apps even if mobile platforms have        
remote: + set -e        
remote: + [ -n Usage: meteor build <output path> [--debug] [--directory] [--server-only]        
remote: Pass `--server-only` to skip building mobile apps, but still build the        
remote:   --server-only       Skip building mobile apps even if mobile platforms have ]        
remote: + SERVER_ONLY_FLAG=--server-only        
remote: + [ -z --server-only ]        
remote: + [ -e /tmp/build_94dcde2cf266af783b2ac6f88761a960/package.json ]        
remote: + npm install        
remote: litmix@ /tmp/build_94dcde2cf266af783b2ac6f88761a960        
remote: ├─┬ babel-runtime@6.23.0         
remote: │ ├── core-js@2.4.1         
remote: │ └── regenerator-runtime@0.10.3         
remote: ├─┬ meteor-node-stubs@0.2.6         
remote: │ ├── asn1.js@4.9.0         
remote: │ ├── assert@1.4.1         
remote: │ ├── Base64@0.2.1         
remote: │ ├── base64-js@1.2.0         
remote: │ ├── bn.js@4.11.6         
remote: │ ├── brorand@1.0.6         
remote: │ ├── browserify-aes@1.0.6         
remote: │ ├── browserify-cipher@1.0.0         
remote: │ ├── browserify-des@1.0.0         
remote: │ ├── browserify-rsa@4.0.1         
remote: │ ├── browserify-sign@4.0.0         
remote: │ ├── browserify-zlib@0.1.4         
remote: │ ├── buffer@4.9.1         
remote: │ ├── buffer-shims@1.0.0         
remote: │ ├── buffer-xor@1.0.3         
remote: │ ├── cipher-base@1.0.3         
remote: │ ├── console-browserify@1.1.0         
remote: │ ├── constants-browserify@1.0.0         
remote: │ ├── core-util-is@1.0.2         
remote: │ ├── create-ecdh@4.0.0         
remote: │ ├── create-hash@1.1.2         
remote: │ ├── create-hmac@1.1.4         
remote: │ ├── crypto-browserify@3.11.0         
remote: │ ├── date-now@0.1.4         
remote: │ ├── des.js@1.0.0         
remote: │ ├── diffie-hellman@5.0.2         
remote: │ ├── domain-browser@1.1.7         
remote: │ ├── elliptic@6.3.2         
remote: │ ├── events@1.1.1         
remote: │ ├── evp_bytestokey@1.0.0         
remote: │ ├── hash.js@1.0.3         
remote: │ ├── http-browserify@1.7.0         
remote: │ ├── https-browserify@0.0.1         
remote: │ ├── ieee754@1.1.8         
remote: │ ├── indexof@0.0.1         
remote: │ ├── inherits@2.0.1         
remote: │ ├── isarray@1.0.0         
remote: │ ├── miller-rabin@4.0.0         
remote: │ ├── minimalistic-assert@1.0.0         
remote: │ ├── os-browserify@0.2.1         
remote: │ ├── pako@0.2.9         
remote: │ ├── parse-asn1@5.0.0         
remote: │ ├── path-browserify@0.0.0         
remote: │ ├── pbkdf2@3.0.9         
remote: │ ├── process@0.11.9         
remote: │ ├── process-nextick-args@1.0.7         
remote: │ ├── public-encrypt@4.0.0         
remote: │ ├── punycode@1.4.1         
remote: │ ├── querystring@0.2.0         
remote: │ ├── querystring-es3@0.2.1         
remote: │ ├── randombytes@2.0.3         
remote: │ ├── readable-stream@2.2.1         
remote: │ ├── ripemd160@1.0.1         
remote: │ ├── sha.js@2.4.8         
remote: │ ├── stream-browserify@2.0.1         
remote: │ ├── string_decoder@0.10.31         
remote: │ ├── timers-browserify@1.4.2         
remote: │ ├── tty-browserify@0.0.0         
remote: │ ├─┬ url@0.11.0         
remote: │ │ └── punycode@1.3.2         
remote: │ ├── util@0.10.3         
remote: │ ├── util-deprecate@1.0.2         
remote: │ └── vm-browserify@0.0.4         
remote: └─┬ ytdl-core@0.10.3         
remote:   ├── html-entities@1.2.0         
remote:   └── sax@1.2.2         
remote: 
remote: + [ -n  ]        
remote: + mktemp -d /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/build-XXXX        
remote: + BUNDLE_DEST=/app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/build-3vPo        
remote: + echo -----> Building Meteor with ROOT_URL: https://litmixes.herokuapp.com        
remote: -----> Building Meteor with ROOT_URL: https://litmixes.herokuapp.com        
remote: + HOME=/app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/meteor-Ikdr/.meteor/meteor build --server https://litmixes.herokuapp.com --server-only --directory /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/build-3vPo        
remote: + mv /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/build-3vPo/bundle /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app        
remote: + rmdir /app/tmp/buildpacks/6c308271b0f0cc3b849f2f9f196b2ddf569c2da726fbf9d2d99ddbdb760488c0a599df87cc380f70e6c5b1753ce5b8fa4c5247bae7786989d8119e10dc0fb6bb/build-3vPo        
remote: + [ -e /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app/programs/server/package.json ]        
remote: + cd /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app/programs/server        
remote: + npm install        
remote: 
remote: > fibers@1.0.15 install /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app/programs/server/node_modules/fibers        
remote: > node build.js || nodejs build.js        
remote: 
remote: make: Entering directory `/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app/programs/server/node_modules/fibers/build'        
remote:   CXX(target) Release/obj.target/fibers/src/fibers.o        
remote: ../src/fibers.cc: In function ‘v8::Handle<v8::String> uni::NewLatin1String(v8::Isolate*, const char*)’:        
remote: ../src/fibers.cc:122:64: warning: ‘static v8::Local<v8::String> v8::String::NewFromOneByte(v8::Isolate*, const uint8_t*, v8::String::NewStringType, int)’ is deprecated (declared at /app/.node-gyp/7.7.4/include/node/v8.h:2542): Use maybe version [-Wdeprecated-declarations]        
remote:    return String::NewFromOneByte(isolate, (const uint8_t*)string);        
remote:                                                                 ^        
remote: ../src/fibers.cc: In function ‘v8::Handle<v8::String> uni::NewLatin1Symbol(v8::Isolate*, const char*)’:        
remote: ../src/fibers.cc:126:64: warning: ‘static v8::Local<v8::String> v8::String::NewFromOneByte(v8::Isolate*, const uint8_t*, v8::String::NewStringType, int)’ is deprecated (declared at /app/.node-gyp/7.7.4/include/node/v8.h:2542): Use maybe version [-Wdeprecated-declarations]        
remote:    return String::NewFromOneByte(isolate, (const uint8_t*)string);        
remote:                                                                 ^        
remote: ../src/fibers.cc: In static member function ‘static uni::FunctionType Fiber::New(const Arguments&)’:        
remote: ../src/fibers.cc:433:99: warning: ‘v8::Local<v8::Object> v8::Function::NewInstance(int, v8::Local<v8::Value>*) const’ is deprecated (declared at /app/.node-gyp/7.7.4/include/node/v8.h:3657): Use maybe version [-Wdeprecated-declarations]        
remote:      return uni::Return(uni::Deref(Isolate::GetCurrent(), tmpl)->GetFunction()->NewInstance(1, argv), args);        
remote:                                                                                                    ^        
remote: ../src/fibers.cc: In static member function ‘static void Fiber::RunFiber(void**)’:        
remote: ../src/fibers.cc:621:14: warning: ‘v8::TryCatch::TryCatch()’ is deprecated (declared at /app/.node-gyp/7.7.4/include/node/v8.h:7616): Use isolate version [-Wdeprecated-declarations]        
remote:      TryCatch try_catch;        
remote:               ^        
remote: ../src/fibers.cc: In static member function ‘static void Fiber::SetPoolSize(v8::Local<v8::String>, v8::Local<v8::Value>, const SetterCallbackInfo&)’:        
remote: ../src/fibers.cc:741:43: warning: ‘v8::Local<v8::Number> v8::Value::ToNumber() const’ is deprecated (declared at /app/.node-gyp/7.7.4/include/node/v8.h:9005): Use maybe version [-Wdeprecated-declarations]        
remote:     Coroutine::pool_size = value->ToNumber()->Value();        
remote:                                            ^        
remote: ../src/fibers.cc: In static member function ‘static void Fiber::Init(v8::Handle<v8::Object>)’:        
remote: ../src/fibers.cc:793:72: warning: ‘bool v8::Object::SetAccessor(v8::Local<v8::String>, v8::AccessorGetterCallback, v8::AccessorSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute)’ is deprecated (declared at /app/.node-gyp/7.7.4/include/node/v8.h:3019): Use maybe version [-Wdeprecated-declarations]        
remote:     fn->SetAccessor(uni::NewLatin1Symbol(isolate, "current"), GetCurrent);        
remote:                                                                         ^        
remote: ../src/fibers.cc:794:87: warning: ‘bool v8::Object::SetAccessor(v8::Local<v8::String>, v8::AccessorGetterCallback, v8::AccessorSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute)’ is deprecated (declared at /app/.node-gyp/7.7.4/include/node/v8.h:3019): Use maybe version [-Wdeprecated-declarations]        
remote:     fn->SetAccessor(uni::NewLatin1Symbol(isolate, "poolSize"), GetPoolSize, SetPoolSize);        
remote:                                                                                        ^        
remote: ../src/fibers.cc:795:84: warning: ‘bool v8::Object::SetAccessor(v8::Local<v8::String>, v8::AccessorGetterCallback, v8::AccessorSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute)’ is deprecated (declared at /app/.node-gyp/7.7.4/include/node/v8.h:3019): Use maybe version [-Wdeprecated-declarations]        
remote:     fn->SetAccessor(uni::NewLatin1Symbol(isolate, "fibersCreated"), GetFibersCreated);        
remote:                                                                                     ^        
remote:   CXX(target) Release/obj.target/fibers/src/coroutine.o        
remote:   CC(target) Release/obj.target/fibers/src/libcoro/coro.o        
remote:   SOLINK_MODULE(target) Release/obj.target/fibers.node        
remote:   COPY Release/fibers.node        
remote: make: Leaving directory `/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app/programs/server/node_modules/fibers/build'        
remote: Installed in `/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app/programs/server/node_modules/fibers/bin/linux-x64-51/fibers.node`        
remote: 
remote: > meteor-dev-bundle@0.0.0 install /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app/programs/server        
remote: > node npm-rebuild.js        
remote: 
remote: internal/child_process.js:319        
remote:     throw errnoException(err, 'spawn');        
remote:     ^        
remote: 
remote: Error: spawn EACCES        
remote:     at exports._errnoException (util.js:1034:11)        
remote:     at ChildProcess.spawn (internal/child_process.js:319:11)        
remote:     at exports.spawn (child_process.js:378:9)        
remote:     at rebuild (/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app/programs/server/npm-rebuild.js:46:5)        
remote:     at Object.<anonymous> (/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app/programs/server/npm-rebuild.js:67:1)        
remote:     at Module._compile (module.js:571:32)        
remote:     at Object.Module._extensions..js (module.js:580:10)        
remote:     at Module.load (module.js:488:32)        
remote:     at tryModuleLoad (module.js:447:12)        
remote:     at Function.Module._load (module.js:439:3)        
remote: 
remote: npm WARN meteor-dev-bundle@0.0.0 No description        
remote: npm WARN meteor-dev-bundle@0.0.0 No repository field.        
remote: npm WARN meteor-dev-bundle@0.0.0 No license field.        
remote: npm ERR! Linux 3.13.0-112-generic        
remote: npm ERR! argv "/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/bin/node" "/tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/bin/npm" "install"        
remote: npm ERR! node v7.7.4        
remote: npm ERR! npm  v4.1.2        
remote: npm ERR! code ELIFECYCLE        
remote: npm ERR! meteor-dev-bundle@0.0.0 install: `node npm-rebuild.js`        
remote: npm ERR! Exit status 1        
remote: npm ERR!         
remote: npm ERR! Failed at the meteor-dev-bundle@0.0.0 install script 'node npm-rebuild.js'.        
remote: npm ERR! Make sure you have the latest version of node.js and npm installed.        
remote: npm ERR! If you do, this is most likely a problem with the meteor-dev-bundle package,        
remote: npm ERR! not with npm itself.        
remote: npm ERR! Tell the author that this fails on your system:        
remote: npm ERR!     node npm-rebuild.js        
remote: npm ERR! You can get information on how to open an issue for this project with:        
remote: npm ERR!     npm bugs meteor-dev-bundle        
remote: npm ERR! Or if that isn't available, you can get their info via:        
remote: npm ERR!     npm owner ls meteor-dev-bundle        
remote: npm ERR! There is likely additional logging output above.        
remote: 
remote: npm ERR! Please include the following file with any support request:        
remote: npm ERR!     /tmp/build_94dcde2cf266af783b2ac6f88761a960/.meteor/heroku_build/app/programs/server/npm-debug.log        
remote:  !     Push rejected, failed to compile Node.js app.        
remote: 
remote:  !     Push failed        
remote: Verifying deploy...        
remote: 
remote: !	Push rejected to litmixes.        
remote: 
To https://git.heroku.com/litmixes.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'https://git.heroku.com/litmixes.git'
```

##### Deployment target

Heroku

**Comment by yourcelf** [comment]
You're using a very old version of the buildpack -- it looks like it's a year and 54 commits behind our master.  In that time we've done a lot of fixes to properly support Meteor v1.4+.  Looking at the commits on your fork, I don't see any changes that you have that aren't incorporated with our master right now.

Could you either merge the latest master from https://github.com/AdmitHub/meteor-buildpack-horse.git into your fork, or switch to this master to see if it builds correctly?

    heroku buildpacks:set https://github.com/AdmitHub/meteor-buildpack-horse.git

**Comment by lpender** [comment]
Thanks! I feel foolish. Totally forgot/didn't notice that I was running a fork.
